fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalOlapScan.java

```diff
@@ -217,7 +217,7 @@ public OlapTable getTable() {
     @Override
     public String toString() {
         return Utils.toSqlString("LogicalOlapScan",
-                "qualified", qualifiedName(),
```

Comment:
why remove quailied name? it is useful if two table name is same but in different db